CODEOWNERS

```diff
@@ -1 +1 @@
-* @acezard @Ldoppea @doubleface
```

Comment:
Why do we remove doubleface?

Reply:
As the connector files are going to another repo, it makes sense to me. @doubleface is assigned on every pull request we make but 99% of the time he is not really concerned by the changes we make. What do you think @doubleface? I mean, this change is mainly to remove the amount of unnecessary notifications you receive